packages/server-wallet/src/models/ledger-request.ts

```diff
@@ -46,56 +75,106 @@ export class LedgerRequest extends Model implements LedgerRequestType {
     await LedgerRequest.query(tx).findById([channelToBeFunded, type]).patch({status});
   }
 
-  static async getPendingRequests(
+  static async getActiveRequests(
     ledgerChannelId: string,
     tx: TransactionOrKnex
-  ): Promise<LedgerRequestType[]> {
-    return LedgerRequest.query(tx).select().where({ledgerChannelId, status: 'pending'});
+  ): Promise<LedgerRequest[]> {
+    return LedgerRequest.query(tx)
+      .select()
+      .where({ledgerChannelId, status: 'pending'})
+      .orWhere({ledgerChannelId, status: 'queued'});
+  }
+
+  static async ledgersWithNewReqeustsIds(tx: TransactionOrKnex): Promise<string[]> {
+    return (
+      await LedgerRequest.query(tx).column('ledgerChannelId').whereNull('lastSeenAgreedState')
```

Comment:
Would you be able to elaborate on how this query compares to a query that reads all ledger requests that have  `queued` or `pending` status?

Reply:
I'm using the fact that `lastSeenAgreedState` will be null if and only if the ledger request has never been involved in a crank before. These requests are therefore the ones that were added in the last round of objective cranks, and therefore these are the ledgers we need to crank.